roles/test_report_send/tasks/reporting/splunk.yml

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# reporting/splunk.yml tasks file for redhatci.ocp.test_report_send
+
+- name: Read content from test report JSON file
+  ansible.builtin.slurp:
+    src: "{{ trs_report_path }}"
+  register: _trs_json_file_content
+
+- name: Decode JSON content of test report file
+  ansible.builtin.set_fact:
+    trs_test_data: "{{ _trs_json_file_content.content | b64decode | from_json }}"
+    trs_default_dt: "{{ now(fmt='%Y-%m-%dT%H:%M:%S') }}"
```

Comment:
Avoid setting facts if only used once, instead set them at the task level as vars